Assets/_Standard Assets/Characters/Scripts/Physics/OpenCharacterController.cs

```diff
@@ -543,6 +546,8 @@ public void OnMoveLoop()
 		/// Default height of the capsule (e.g. for resetting it).
 		/// </summary>
 		public float defaultHeight { get; private set; }
+		
+		public bool slowAgainstWalls { get { return m_SlowAgainstWalls; } }
```

Comment:
Good that you added this! We like to ensure all public properties have documentation, though, so please add it here just like the other properties.